src/components/EventForm/index.js

```diff
@@ -150,12 +151,76 @@ export default function EventForm({
         onValuesChange={() => {}}
         onFinish={onFinish}
       >
-        <Form.Item required name="date" label="Date">
-          <DatePicker />
-        </Form.Item>
-        <Form.Item required name="title" label="Event title">
-          <Input placeholder="What happened?" />
-        </Form.Item>
+        
+
+const layout = {
+  labelCol: {
+    span: 8,
+  },
+  wrapperCol: {
+    span: 16,
+  },
+};
+
+const tailLayout = {
+  wrapperCol: {
+    offset: 8,
+    span: 16,
+  },
+};
+
+const MyForm = () => {
+  const onFinish = (values) => {
+    console.log('Success:', values);
+  };
+
+  const onFinishFailed = (errorInfo) => {
+    console.log('Failed:', errorInfo);
+  };
+
+  return (
+    <Form>
+      {...layout}
+      name="basic"
+      initialValues={{
+        remember: true,
+      }}
+      onFinish={onFinish}
+      onFinishFailed={onFinishFailed}
+    >
+      <Form.Item
+        label="Date"
+        name="date"
+        rules={[
+          {
+            required: true,
```

Comment:
This isn't a valid change @HariHaran1214 . you have declared variables and new component inside the render part. This breaks. Please make sure your changes work fine in local and then raise a valid PR